### ISSUE TITLE
Add Screenshot to Readme

### ISSUE THREAD

**Issue by skellock** [infinitered/ignite#25]
It'd be nice to have a screenshot of what we're building here.  Just for context.

**Comment by markrickert** [comment]
Might want to wait on this for a bit simply because the main screen seems to be in constant flux right now.

**Comment by jamonholmgren** [comment]
Here's one that will work, @skellock:

<img width="250" alt="screen shot 2016-03-30 at 2 47 56 pm" src="https://cloud.githubusercontent.com/assets/1479215/14158708/6edce380-f686-11e5-9cd8-9939c0ce2390.png">

**Comment by GantMan** [comment]
womp womp

**Comment by kevinvangelder** [comment]
I think we should be good to go on this once #110 is finished. A quick gif clicking around to the various sections of the app would be pretty nifty.

**Comment by GantMan** [comment]
agreed, now that the front screen is nice looking.

**Comment by GantMan** [comment]
a sized down version of https://github.com/infinitered/ignite/pull/134 would be good, once the PR is accepted.
